Example/iOS Example/Info.plist

```diff
@@ -33,6 +33,7 @@
 		<string>UIInterfaceOrientationPortrait</string>
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
+		<string>UIInterfaceOrientationPortraitUpsideDown</string>
```

Comment:
Fixes a warning when running on device.

Reply:
That's odd. Normally `PortraitUpsideDown` is only suggested on iPads.